File: converter-dotnet-core-console-vb/SimpleConversionConverter.vb
Imports ceTe.DynamicPDF.Conversion

Class SimpleConversionConverter

    Public Shared Sub Run()
        TextConvert()
        RTFConvert()
        WordConvert()
        PowerPointConvert()
        ExcelConvert()
        ImageConvert()
        HtmlConvert()
    End Sub

    Private Shared Sub TextConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.txt"), Program.GetOutputDocPath("simple-txt-output.pdf"))
    End Sub

    Private Shared Sub RTFConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.rtf"), Program.GetOutputDocPath("simple-rtf-output.pdf"))
    End Sub

    Private Shared Sub WordConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.doc"), Program.GetOutputDocPath("simple-doc-output.pdf"))
    End Sub

    Private Shared Sub PowerPointConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.pptx"), Program.GetOutputDocPath("simple-pptx-output.pdf"))
    End Sub

    Private Shared Sub ExcelConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.xlsx"), Program.GetOutputDocPath("simple-xlsx-output.pdf"))
    End Sub

    Private Shared Sub ImageConvert()
        Converter.Convert(Program.GetResourcePath("DocumentA.gif"), Program.GetOutputDocPath("simple-gif-output.pdf"))
    End Sub

    Private Shared Sub HtmlConvert()
        Converter.Convert(Program.GetResourcePath("moby-dick.html"), Program.GetOutputDocPath("simple-html-output.pdf"))
    End Sub

End Class

File: converter-dotnet-core-console-cs/SimpleConversionConverter.cs
using ceTe.DynamicPDF.Conversion;

namespace converter_dotnet_core_console_cs
{
    class SimpleConversionConverter
    {
        public static void Run()
        {
            TextConvert();
            RTFConvert();
            WordConvert();
            PowerPointConvert();
            ExcelConvert();
            ImageConvert();
            HtmlConvert();
        }

        private static void TextConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.txt"), Program.GetOutputDocPath("simple-txt-output.pdf"));
        }

        private static void RTFConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.rtf"), Program.GetOutputDocPath("simple-rtf-output.pdf"));
        }

        private static void WordConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.doc"), Program.GetOutputDocPath("simple-doc-output.pdf"));
        }

        private static void PowerPointConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.pptx"), Program.GetOutputDocPath("simple-pptx-output.pdf"));
        }
        private static void ExcelConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.xlsx"), Program.GetOutputDocPath("simple-xlsx-output.pdf"));
        }

        private static void ImageConvert()
        {
            Converter.Convert(Program.GetResourcePath("DocumentA.gif"), Program.GetOutputDocPath("simple-gif-output.pdf"));
        }

        private static void HtmlConvert()
        {
            Converter.Convert(Program.GetResourcePath("moby-dick.html"), Program.GetOutputDocPath("simple-html-output.pdf"));
        }


    }
}
